Makefile

```diff
@@ -2,6 +2,11 @@
 SHELL = /bin/bash
 NODE_BINDIR = ./node_modules/.bin
 export PATH := $(NODE_BINDIR):$(PATH)
+LOGNAME := $(shell logname)
+
+# adding the name of the user's login name to the template file, so that
+# on a multi-user system several users can run this without interference
+TEMPLATE_POT := /tmp/template-$(LOGNAME).pot
```

Comment:
I'd put `TEMPLATE_POT ?= /tmp/...` so that users could override this variable also.